library/oneview_network_set.py

```diff
@@ -140,18 +141,29 @@ def execute_module(self):
             return self.resource_absent()
 
     def __present(self):
+
+        bandwidth = self.data.pop('bandwidth', None)
         scope_uris = self.data.pop('scopeUris', None)
         self.__replace_network_name_by_uri(self.data)
         result = self.resource_present(self.RESOURCE_FACT_NAME)
+
+        if bandwidth:
+            if self.__update_connection_template(bandwidth):
+                if not result['changed']:
+                    result['changed'] = True
+                    result['msg'] = self.MSG_UPDATED
```

Comment:
Try adding facts in the return response.
```
return['ansible_facts']['connection_template'] = response
```
response is the return value of '__update_connection_template' method(None/connection_template.data)

Reply:
I have added connection_template to ansible_facts and updated the tests accordingly.